### PR TITLE
add accelerating sdlc workshop

### PR DESCRIPTION
**Pull request by tiffanywang3**

### CODE REVIEW

File: content/20_weaveworks_prerequisites/14_install_fluxctl.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
++++
+
+## Install Fluxctl
+
+[Fluxctl](https://docs.fluxcd.io/en/1.17.1/references/fluxctl.html) is a CLI tool that is able to talk to [Weave Flux](https://github.com/fluxcd/flux).
+
+Install by running this command:
+
+```sh
+curl -sL https://fluxcd.io/install | sh
```
Comment (richardcase) [inline content/20_weaveworks_prerequisites/14_install_fluxctl.md]: Do you want a `\` on these commands?

File: content/50_workshop_5_accelerating_sdlc/00_prerequisites.md/30_install_fluxctl.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+title = "Install fluxctl"
+chapter = false
+weight = 30
+
++++
+
+We will be using [Fluxctl](https://docs.fluxcd.io/en/1.17.1/references/fluxctl.html), a CLI tool that is able to talk to [Weave Flux](https://github.com/fluxcd/flux) to immediately apply and deploy changes we make to our repository.
+
+```sh
+curl -sL https://fluxcd.io/install | sh
```
Comment (richardcase) [inline content/50_workshop_5_accelerating_sdlc/00_prerequisites.md/30_install_fluxctl.md]: Same as above with the command continuation `\`

File: content/50_workshop_5_accelerating_sdlc/00_prerequisites.md/60_add_iam_role.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
++++
+title = "Add the Required IAM Role"
+chapter = false
+weight = 60
+
++++
+
+Creating and using EKS clusters in AWS requires specific IAM roles for the user creating and accessing EKS clusters.
+
+- Switch to the **AWS Console** (You can open the console from the "Team Dashboard")
```
Comment (richardcase) [inline content/50_workshop_5_accelerating_sdlc/00_prerequisites.md/60_add_iam_role.md]: I know you didn't create this. Just for the workshop, in Cloud9 if you click the buttong that is a A in a  circle next to shar in the top right, you can then click on `Manage EC2 Instance`

File: content/50_workshop_5_accelerating_sdlc/40_progressive_delivery/03_canary_rollback.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+View Flagger logs with:
+
+```sh
+kubectl -n appmesh-system logs deployment/flagger -f | jq .msg
+```
+
+Exec into the tester pod and generate HTTP 500 errors:
+
+```sh
+kubectl -n demo exec -it $(kubectl -n demo get pods -o name | grep -m1 flagger-loadtester | cut -d'/' -f 2) bash
```
Comment (richardcase) [inline content/50_workshop_5_accelerating_sdlc/40_progressive_delivery/03_canary_rollback.md]: Do we need to split this section into 2 code blocks? One to exec and then another one for the hey commands? The hey commangs could be `&&` together?

File: content/50_workshop_5_accelerating_sdlc/50_deployment_strategies/01_recreate.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+
+{{<mermaid>}}
+graph LR;
+    A{Deployment} --> B(Replica 1, V1)
+    A --> C(Replica 2, V1)
+{{</mermaid>}}
+
+The recreate strategy will terminate both replicas. During this time, the service is down and will not be able to handle traffic:
+
+{{<mermaid>}}
```
Comment (richardcase) [inline content/50_workshop_5_accelerating_sdlc/50_deployment_strategies/01_recreate.md]: I didn't know about mermaid!

**Review by richardcase — APPROVED** [review]
A great workshop.

File: content/50_workshop_5_accelerating_sdlc/00_prerequisites.md/30_install_fluxctl.md (hunk 2 above)
Comment (tiffanywang3) [reply content/50_workshop_5_accelerating_sdlc/00_prerequisites.md/30_install_fluxctl.md]: have been forgetting to mention that I've fixed the comments, thank you Richard!

**Pull request merged**